theme-utils.mjs

```diff
@@ -1536,26 +1537,14 @@ async function validateThemes( themes, { format, color, tableWidth } ) {
 		}
 
 		const styleCss = await fs.promises.readFile( styleCssPath, 'utf-8' );
-		const themeRequires = getThemeMetadata( styleCss, 'Requires at least' );
+		const themeRequires = getThemeMetadata( styleCss, 'Requires at least', true );
 		const wpVersion = themeRequires
 			? `${ themeRequires }.0`.split( '.', 2 ).join( '.' )
 			: undefined;
-		const isSupportedWpVersion = wpVersion && semver.gte( `${ wpVersion }.0`, '5.9.0' )
+		const isSupportedWpVersion = wpVersion && semver.valid( `${ wpVersion }.0` ) && semver.gte( `${ wpVersion }.0`, '5.9.0' )
```

Comment:
```suggestion
		const hasThemeJsonSupport = wpVersion && semver.valid( `${ wpVersion }.0` ) && semver.gte( `${ wpVersion }.0`, '5.9.0' )
```

Reply:
fixed in [75fd4e9](https://github.com/Automattic/themes/pull/8068/commits/75fd4e9f2b2a9ad3802814050e10b41ce372a783)